code/modules/projectiles/projectile/bullets.dm

```diff
@@ -143,16 +143,24 @@
 	armor_penetration = 20
 	agony = 31
 
-/obj/item/projectile/bullet/pistol/medium/smg/hollowpoint
+/obj/item/projectile/bullet/pistol/medium/smg/rubber
 	fire_sound = 'sound/weapons/gunshot/gunshot_smg.ogg'
-	damage = 34 //10mm hollowpoint
+	damage = 1.5 //10mm rubber
 	armor_penetration = 12
-	agony = 25
+	agony = 57
```

Comment:
This is an enormous amount of agony damage per shot and keep in mind that smgs can be fired in burst mode